File: main/src/main/java/net/fwitz/math/main/binary/dual/functions/exponential/DualLogZ.java
package net.fwitz.math.main.binary.dual.functions.exponential;

import net.fwitz.math.binary.dual.DualNumber;
import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot;

public class DualLogZ {
    public static void main(String[] args) {
        new DualNumberFunctionPlot("log(z)")
                .computeFn(DualNumber::log)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/dual/functions/exponential/DualLogZ.kt
package net.fwitz.math.main.binary.dual.functions.exponential

import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot

object DualLogZ {
    @JvmStatic
    fun main(args: Array<String>) = DualNumberFunctionPlot("log(z)")
        .computeFn { z -> z.log }
        .render()
}